adobe_xd/lib/page_link.dart

```diff
@@ -30,11 +30,11 @@ enum LinkTransition {
 }
 
 class PageLinkInfo {
-  final Function pageBuilder;
+  final Function? pageBuilder;
   final LinkTrigger trigger;
   final LinkTransition transition;
-  final Curve ease;
-  final double duration;
+  final Curve? ease;
+  final double? duration;
```

Comment:
Similarly this being nullable is leading to a bunch of !'s, we should mark this as required, or assign a default here, `Duration.zero`?

Reply:
ok I did default `duration = 0` and `ease = Curves.easeOut`